packages/config/src/BaseFeature.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  Autocomplete, Search, SmartCollection, Recommendations, Content
+} from '@findify/sdk/types/request'
+import { ProductType } from './ProductType';
+import { Breakpoints } from './Breakpoints';
+
+export type RequestParams = {
+  Autocomplete: Autocomplete.Params,
+  Search: Search.Params,
+  SmartCollection: SmartCollection.Params,
+  Recommendations: Recommendations.Params,
+  Content: Content.Params,
+}
+
+export type BaseFeature<T extends keyof RequestParams> = {
+  /**
+   * Should widget send request right after creation
+   * Helpful if widget is created manually and you need to set
+   * default request properties
+   */
+	disableAutoRequest: boolean
+
+  /**
+   * Default widget request params
+  */
+  meta: RequestParams[T]
```

Comment:
@Bitaru mb separate meta and config ? btw, If we going to share this type with compilation server - it will not work, as compilation server doesn't know about meta

Reply:
@timakoder This field used in 2 places to create default request. I think its kinda useful for customizations to set limit

Reply:
@Bitaru alright, lets mb then create separate field e.g. `defaultRequestParams` ? meta is not descriptive at all :)

Reply:
@timakoder do we not setting autocomplete item_limit and suggestion_limit in autocomplete?

Reply:
+ 1